config/initializers/raven.rb

```diff
@@ -1,6 +1,7 @@
 Raven.configure do |config|
   config.silence_ready = true
   config.current_environment = HostingEnvironment.environment_name
+  config.release = "#{HostingEnvironment.environment_name}@#{ENV['SHA']}"
```

Comment:
I'm not sure if we should prefix this. In Sentry you can say "Resolve in next release", which means the error will only re-alert if it happens in a next version. If the version is hosting environment-specific, it won't work across environments.